src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const FETCH_ROCKETS = ' FETCH_ROCKETS';
+const initialState = [];
+
+export const fetchRockets = createAsyncThunk(
+  FETCH_ROCKETS,
+  async (_, { dispatch }) => {
+    const response = await fetch('https://api.spacexdata.com/v3/rockets');
+    const data = await response.json();
+    const rockets = data.map((item) => ({
+      id: item.rocket_id,
+      name: item.rocket_name,
+      type: item.rocket_type,
+      image: item.flickr_images[0],
+    }));
+    dispatch({
+      type: FETCH_ROCKETS,
+      payload: rockets,
+    });
+    console.log(rockets);
```

Comment:
[Optional] Hi, while debugging it is okay to use console.log statements, but it is better to remove it before pushing it to repository as it unnecessary log the data.

Reply:
I added it to consoled log the fetched data, let me remove it